src/Chainweb/Version.hs

```diff
@@ -652,7 +652,8 @@ useLegacyCreationTimeForTxValidation
     :: ChainwebVersion
     -> BlockHeight
     -> Bool
-useLegacyCreationTimeForTxValidation Mainnet01 _ = True
+useLegacyCreationTimeForTxValidation Mainnet01 h = h < 449940 -- ~ 2020-04-03T00:00:00Z
+useLegacyCreationTimeForTxValidation Testnet04 h = h < 286231 -- 2020-04-02T02:00:00Z
```

Comment:
6.2 days from now, and it's currently 21:00 UTC. A quarter of a day is ~5 hours, so 02:00 is correct.

Reply:
4/2 1:56 AM